pages/linux/agetty.md

```diff
@@ -0,0 +1,30 @@
+# agetty
+
+> Alternative `getty`: Open a `tty` port, prompt for a login name, and invoke the `/bin/login` command.
+> Note: The baud rate is the speed of data transfer between a terminal and a device over a serial connection.
+> It is normally invoked by `init`.
```

Comment:
I would suggest switching the order of the last two lines since now the first is about the command, the second about something else and the third about the command again.
```suggestion
> Alternative `getty`: Open a `tty` port, prompt for a login name, and invoke the `/bin/login` command.
> It is normally invoked by `init`.
> Note: the baud rate is the speed of data transfer between a terminal and a device over a serial connection.
```